exomedepth/Dockerfile

```diff
@@ -0,0 +1,20 @@
+FROM r-base:4.1.2
+
+# install curl and libcurl
+RUN apt-get update && apt-get install --no-install-recommends -y curl=7.82.0-2 \
+  && apt-get --no-install-recommends -y install libcurl4-openssl-dev=7.82.0-2 \
```

Comment:
You can combine the two `apt-get install` commands and just list the packages after each other.